env0/resource_template.go

```diff
@@ -88,6 +88,11 @@ func resourceTemplate() *schema.Resource {
 				Description: "if specified, will only retry (on destroy) if error matches specified regex",
 				Optional:    true,
 			},
+			"github_installation_id": {
+				Type:        schema.TypeInt,
+				Description: "description for the template",
```

Comment:
Please fix the description for this attribute

Reply:
b8eb7e24e86f888217bb444ede4084e39046e515